proto/autokitteh/values/v1/values.proto

```diff
@@ -82,11 +82,11 @@ message Function {
   // executor identifies who owns the execution of the function.
   // essentially, where the function is defined. that entity
   // is responsible of the execution of the function.
-  string executor_id = 1 [(buf.validate.field).string.min_len = 1];
+  string executor_id = 1;
```

Comment:
allowed empty?

Reply:
Yes, multiple places create values without a xid and without a desc - according to Itay that's ok and shouldn't change, so the check is outdated.